README.md

```diff
@@ -1,13 +1,13 @@
-# Product Opener
+<img height='175' src="https://static.openfoodfacts.org/images/svg/openfoodfacts-logo-en.svg" align="left" hspace="1" vspace="1">
```

Comment:
Does html work in readme files on github?
It may be better to use markdown: https://guides.github.com/features/mastering-markdown/